src/zkevm_specs/sig_circuit.py

```diff
@@ -0,0 +1,120 @@
+from typing import List, NamedTuple
+from .util import FQ, RLC, Word, ECDSAVerifyChip, KeccakTable, is_circuit_code
+from eth_keys import KeyAPI  # type: ignore
+from eth_utils import keccak
+
+
+class Row:
```

Comment:
This circuit comes from `SignVerifyChip` in tx_circuit and added some columns for `sig_table`